cmake/onnxruntime_csharp.cmake

```diff
@@ -34,6 +34,10 @@ if (onnxruntime_USE_NUPHAR)
   STRING(APPEND CSHARP_PREPROCESSOR_DEFINES "USE_NUPHAR;")
 endif()
 
+if (onnxruntime_USE_STVM)
```

Comment:
Question for maintainers of ONNX-RT, we used `STVM` to contrast with `Nuphar` when building this EP, should we consider renaming this in the future/consolidating into a single EP?